src/components/Jam.jsx

```diff
@@ -46,34 +48,28 @@ export default function Jam() {
   const [collaborativeOperationRunning, setCollaborativeOperationRunning] = useState(false)
   const [schedule, setSchedule] = useState(null)
 
+  // Returns one fresh address for each requested mixdepth.
   const getNewAddresses = useCallback(
-    (count, mixdepth) => {
+    (mixdepths) => {
+      if (mixdepths.length !== 3) {
+        throw new Error('Can only handle 3 destination addresses for now.')
+      }
       if (!walletInfo) {
         throw new Error('Wallet info is not available.')
       }
```

Comment:
Hmm.. is this check _really_ necessary?
It feels not right.
e.g. what about:

```suggestion
      if (!walletInfo) {
        throw new Error('Wallet info is not available.')
      }
      if (mixdepths.length > walletInfo.data.display.walletinfo.accounts) {
        throw new Error('Can only provide one address per account.')
      }
      if (mixdepths.length <= 3) {
        throw new Error('Can only handle at least 3 destination addresses for now.')
      }
```

Also, nothing is stopping anybody from providing [0,0,0], which might be a better thing to check, right?

Reply:
The reason why I put the check in is to avoid anyone thinking they can just do something like:

```
INTERNAL_DEST_ACCOUNTS = [0, 1, 2, 3]
```

because the UI doesn't support anything else than exactly 3 destination addresses yet. I didn't want to put in the work to make this fully dynamic quite yet, that's why I added this check. Basically an assert. But I agree it's not super clean in this place.

Reply:
Should we remove the check and rely on the comment that says `INTERNAL_DEST_ACCOUNTS` must have length 3?